pagopa/io-app
# feat: [IOBP-373] New wallet details screen adaption

## ⚠️ This PR depends on #5155 and https://github.com/pagopa/io-dev-api-server/pull/319

## Short description
This PR adapts the wallet details page with the new wallet handling. The aim of this PR is also to create a unique screen page that is used for each payment method onboarded by the user instead having a detail page different for each payment method type.

## List of changes proposed in this pull request
- Updated swagger onboading definitions;
- Created the folder structure `walletV3/details` in order to have his own flow about the wallet details page with the actions, sagas and reducer;
- Adapted new wallet details screen with the new payment gateway
- Added the possibility to load wallet information remotely direct into the wallet detail page instead getting it into store

## How to test
- Checkout this PR on the `io-dev-api-server` https://github.com/pagopa/io-dev-api-server/pull/319
- Try to onboard a new payment method into "New Wallet Playground" section, then when you select the success outcome, you should be able to see the payment method details correctly

## Preview

https://github.com/pagopa/io-app/assets/34343582/c2a05ef9-58a3-4950-8157-15dc2076fc85
